saleor/product/utils/availability.py

```diff
@@ -144,6 +148,12 @@ def get_product_availability(
     )
 
     is_on_sale = product.is_visible and discount is not None
+    if country is not None:
```

Comment:
Instead of `if else` condition, You could do something like this:
`country = country or DEFAULT_COUNTRY`
and you will have only a single call of `is_product_in_stock`

Reply:
but this is only suggestion :)